pytest.ini

```diff
@@ -3,3 +3,4 @@ addopts = -x
 DJANGO_SETTINGS_MODULE=ideascube.settings
 norecursedirs = debian
                 .pybuild
+usefixtures = storage_dir
```

Comment:
Does this mean it gets used for every single unit tests?

If so, why did you pass it to some test functions as a parameter?